### PR TITLE
Update demos to use latest web3auth packages

### PR DESCRIPTION
**Pull request by shahbaz17**

### CODE REVIEW

File: demo/auth0-react-app/app/package.json [hunk 1]
```diff
@@ -31,9 +31,11 @@
...
   "browserslist": {
     "production": [
-      ">0.2%",
-      "not dead",
-      "not op_mini all"
+      "chrome >= 67",
+      "edge >= 79",
+      "firefox >= 68",
+      "opera >= 54",
+      "safari >= 14"
```
Comment (yashovardhan) [inline demo/auth0-react-app/app/package.json:38]: we need to document this in the docs.

**Review by yashovardhan — CHANGES_REQUESTED** [review]
The changes are good, can we have readme for each project telling everyone about how to build each demo? Particularly around using lerna bootstrap and nrb before running these demos as well.

Also there should be some explanation around why we have made changes in the package.json - the bigint error.

File: demo/auth0-react-app/app/package.json (hunk 1 above)
Comment (shahbaz17) [reply demo/auth0-react-app/app/package.json:38]: As discussed. We'll be adding this to a separate troubleshooting page and we can even create a GitHub Discussion around it for now.

**Comment by shahbaz17** [comment]
> The changes are good, can we have readme for each project telling everyone about how to build each demo? Particularly around using lerna bootstrap and nrb before running these demos as well.

READMEs updated. 
@yashovardhan

**Review by yashovardhan — APPROVED** [review]
LGTM

**Review by chaitanyapotti — APPROVED** [review]
lgtm

**Pull request merged**